helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -88,6 +92,19 @@ spec:
           value: {{- if or .Values.django.ingress.activateTLS .Values.django.nginx.tls.enabled }} "True" {{- else }} "False" {{- end }}
         - name: DD_CSRF_COOKIE_SECURE
           value: {{- if or .Values.django.ingress.activateTLS .Values.django.nginx.tls.enabled }} "True" {{- else }} "False" {{- end }}
+        livenessProbe:
+          httpGet:
+            path: /login?next=/
+            port: http
+            {{- if .Values.django.nginx.tls.enabled }}
```

Comment:
I thought this probe was adressing uwsgi without going through nginx? why care about `django.nginx.tls.enabled` then?

Reply:
You're right, i'll remove it